apps/meteor/client/views/room/contextualBar/RoomMembers/RoomMembers.tsx

```diff
@@ -86,6 +87,32 @@ const RoomMembers = ({
 
 	const useRealName = useSetting('UI_Use_Real_Name', false);
 
+	const { counts, titles } = useMemo(() => {
+		const owners = members.filter((member) => member.roles?.includes('owner'));
+		const moderators = members.filter((member) => !member.roles?.includes('owner') && member.roles?.includes('moderator'));
+		const normalMembers = members.filter((member) => !member.roles?.includes('owner') && !member.roles?.includes('moderator'));
+
+		const counts = [];
```

Comment:
Isn't `counts` the same as `members.length`?

Reply:
Its actually an array whose each item tells the number of items in each group - owner, moderators, members.